proto/commons/massa/model/v1/commons.proto

```diff
@@ -13,6 +13,17 @@ option php_namespace = "Com\\Massa\\Model\\V1";
 option ruby_package = "Com::Massa::Model::V1";
 option swift_prefix = "MMODEL";
 
+// Massa error
+message Error {
+  // The error code
+  int32 code = 1;
+  // A developer-facing error message, which should be in English
+  string message = 2;
+}
+
+// Empty
```

Comment:
IIRC this type won't be used anymore on the ABI side when we will merge.
do you use it in any API?

Reply:
don't mind, I've found where you use it.
I though you use dedicated empty types?

Reply:
It's used in the API, so we could keep It at the moment. We created our Empty and Error type to not rely on google ones